assets/src/infrastructure/ui/display/steps/Step.tsx

```diff
@@ -3,17 +3,37 @@ import classNames from 'classnames';
 
 import { StepProps } from './types';
 
-const Step: React.FC<StepProps> = ({ active, className, disabled, icon: Icon, showStepNumber, stepNumber, title }) => {
-	const newClassName = classNames('ee-steps__step', className, {
-		active,
-		disabled,
-	});
+const Step: React.FC<StepProps> = ({
+	active,
+	className,
+	description = '',
+	disabled,
+	icon: Icon,
+	showStepNumber,
+	stepNumber,
+	title,
+	vertical = false
+}) => {
+	const newClassName = classNames(
```

Comment:
if we'd go this way, than `enhancedClassName` might be a better name, `newClassName` isn't describing well what it is actually doing